src/main/java/frc/robot/Constants.java

```diff
@@ -196,7 +196,9 @@ public static final class TargetPoses {
 
         public static double BLUE_SCORING_X = 2.0;
         public static double BLUE_SCORING_Y = 0.54;
+        
         public static double SCORING_SPACING = 0.559;
+        public static double SUBSTATION_SPACING = 0.5;
 
         public static double BLUE_SUBSTATION_X = 14.699;
         public static double BLUE_SUBSTATION_Y = 7.283;
```

Comment:
Subtract 0.5 from the BLUE_SUBSTATION_Y value. This constant should define the lower substation Y position, and at the moment I think it defines the upper one.